src/logic/tokens/utils/formatAmount.ts

```diff
@@ -39,3 +39,10 @@ export const formatAmount = (number) => {
 
   return numberFloat
 }
+
+export const formatAmountInUsFormat = (amount: string): string => {
+  const options = { style: 'currency', currency: 'USD', minimumFractionDigits: 2, maximumFractionDigits: 8 }
+  const numberFormat = new Intl.NumberFormat('en-US', options)
```

Comment:
options and formatter could be defined outside this function, you don't need to recreate them every function call

Reply:
fixed